admins/pageflow/user.rb

```diff
@@ -120,7 +120,8 @@ module Pageflow
 
     collection_action 'delete_me', :title => I18n.t('pageflow.admin.users.account'), :method => [:get, :delete] do
       if request.delete?
-        if current_user.destroy_with_password(params.require(:user)[:current_password])
+        if Pageflow.config.authorize_user_deletion.call == true &&
```

Comment:
Let's use `authorized?` here and move the call of the configured lambda to the `AbilityMixin` (and later on to a policy).